.travis.yml

```diff
@@ -19,9 +19,9 @@ matrix:
     - jdk: "oraclejdk7" # this will also deploy a uberjar to s3 at some point
       env: PROFILE="-Dhadoop.profile=1"
     - jdk: "openjdk7"
-      env: PROFILE="-P!include-yarn -Dhadoop.version=2.0.0-alpha"
-    - jdk: "oraclejdk7"
-      env: PROFILE="-Dhadoop.version=2.2.0"
+      env: PROFILE="-Dhadoop.version=2.4.0"
```

Comment:
As per mailing list discussion, we agreed to support hadoop 2.3.0.
But it seems that the yarn tests are not working with 2.3.0, so if you want we can set the version to 2.3.0 when the tests are fixed.